src/geo/crs/CRS.Earth.js

```diff
@@ -12,22 +13,14 @@ import * as Util from '../../core/Util';
  */
 
 export var Earth = Util.extend({}, CRS, {
-	wrapLng: [-180, 180],
+	wrapLng: wrapLng,
 
 	// Mean Earth Radius, as recommended for use by
```

Comment:
Move non-docstring comments about the mean earth radius to `CRS.Earth.distance`.